rest_api/operator_apis/dns-operator-openshift-io-v1.adoc

```diff
@@ -180,8 +180,8 @@ Type::
 Description::
 +
 --
-servers is a list of DNS resolvers that provide name query delegation for one or more subdomains outside the scope of the cluster domain. If servers consists of more than one Server, longest suffix match will be used to determine the Server. 
- For example, if there are two Servers, one for "foo.com" and another for "a.foo.com", and the name query is for "www.a.foo.com", it will be routed to the Server with Zone "a.foo.com". 
+servers is a list of DNS resolvers that provide name query delegation for one or more subdomains outside the scope of the cluster domain. If servers consists of more than one Server, longest suffix match will be used to determine the Server.
```

Comment:
Same comment about capitalized "server".

"..., the longer suffix match is used to determine..."